blog.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt A Dog - Blog</title>
+    <link rel="stylesheet" type="text/css" href="./styles/style.css">
+    <script src="https://kit.fontawesome.com/49a0808614.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+      <header class="header">
+        <img src="./images/adoptadog-logo.svg" alt="sitting dog sillouette logo">
+        <h3>Seattle's premiere dog adoption agency</h3>
+        <i class="fas fa-shopping-cart"></i>
+        <button class="checkout" type="button">
+            Checkout
+        </button>
+      </header>
+      <nav class="bar">
+        <ul>
+            <li><a href="default.asp">Home</a></li>
+            <li><a href="dogs.html">Dogs</a></li>
+            <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+      <h2>Adoptadog Blog</h2>
+      <section class="main-blog-container">   
```

Comment:
nice! This will serve you well with stying (and some of the other work later on) great use of the - for class names